app/components/shared/qualification_row_component.html.erb

```diff
@@ -10,18 +10,13 @@
   </td>
   <td class="govuk-table__cell">
     <%= qualification.award_year %>
-    <% if editable %>
-      <%= govuk_link_to support_interface_application_form_edit_other_qualification_award_year_path(qualification_id: qualification.id) do %>
-        Change <span class="govuk-visually-hidden">award year</span>
-      <% end %>
-    <% end %>
   </td>
   <td class="govuk-table__cell">
     <%= render QualificationGradeComponent.new(qualification: qualification) %>
-    <% if editable %>
-      <%= govuk_link_to support_interface_application_form_edit_other_qualification_grade_path(qualification_id: qualification.id) do %>
-        Change <span class="govuk-visually-hidden">grade</span>
-      <% end %>
-    <% end %>
+  </td>
+  <td class="govuk-table__cell">
+  <% if editable %>
+    <%= govuk_link_to('Change', support_interface_application_form_edit_other_qualification_path(id: qualification.id)) %>
+  <% end %>
```

Comment:
Is this shared component used in what interfaces? Provider and support?

Reply:
Yes, provider and support. There's a comment at the top of the file: https://github.com/DFE-Digital/apply-for-teacher-training/blob/3a54475e19196fce8e9d89d8327699b6314a472f/app/components/shared/qualifications_table_component.rb#L1